repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/actor.py

```diff
@@ -26,16 +29,43 @@ def process(self):
         signed_pkgs = InstalledRedHatSignedRPM()
         unsigned_pkgs = InstalledUnsignedRPM()
 
+        env_vars = self.configuration.leapp_env_vars
+        # if we start upgrade with LEAPP_DEVEL_RPMS_ALL_SIGNED=1, we consider
+        # all packages to be signed
+        all_signed = [
+            env
+            for env in env_vars
+            if env.name == 'LEAPP_DEVEL_RPMS_ALL_SIGNED' and env.value == '1'
+        ]
+
+        def some_rhsigs_keys_in_pkg_pgpsig(pkg):
+            return any(key in pkg.pgpsig for key in RH_SIGS)
+
+        def has_gpg_pubkey(pkg):
```

Comment:
```suggestion
        def is_gpg_pubkey(pkg):
```
It checks if the package name is gpg-pubkey

Reply:
you're right @vinzenz